test-lit/semantics/allocate10.f90

```diff
@@ -3,6 +3,9 @@
 !TODO: mixing expr and source-expr?
 !TODO: using subcomponent in source expressions
 
+! RUN: not %flang -fdebug-resolve-names -fparse-only %s 2>&1 | FileCheck --check-prefixes=C942a,ERROR %s
+
+
 subroutine C939_C942a_C945b(xsrc1a, xsrc1c, xsrc0, xsrc2a, xsrc2c, pos)
 ! C939: If an allocate-object is an array, either allocate-shape-spec-list shall
```

Comment:
@kiranchandramohan Notice the space between the comment character and the standard reference.